Readme.md

```diff
@@ -45,41 +45,50 @@ Prior to using Explorer for Endevor, ensure that you meet the following prerequi
 
 Explorer for Endevor is included with Eclipse Che version 7.6.0 and above. Check [here](https://www.eclipse.org/che/docs/che-7/introduction-to-eclipse-che/) for more information.
 
-
 ## **User Guide**
 
 Create a Host in Explorer for Endevor to enable it to access the repository you want to explore.
```

Comment:
Create a profile in Explorer for Endevor to access Endevor inventory locations. If you have an Endevor Plug-in for Zowe CLI profile created, it will appear in the tree and you can access inventory locations immediately.

Reply:
I've added this + more to the intro and deleted the old "create a host" sentence